packages/sdk/tasks/one-time/stella-upgrade.ts

```diff
@@ -0,0 +1,89 @@
+import { TransactionReceipt } from "@ethersproject/abstract-provider";
+import { task, types } from "hardhat/config";
+
+import { FuseFeeDistributor } from "../../lib/contracts/typechain/FuseFeeDistributor";
+
+task("plugin:deploy", "Deploy ERC4626 Strategy")
+  .addParam("creator", "Deployer Address", "deployer", types.string)
+  .addOptionalParam(
+    "otherParams",
+    "other params that might be required to construct the strategy",
+    undefined,
+    types.string
+  )
+  .setAction(async (taskArgs, { ethers, deployments }) => {
+    // @ts-ignore
+    const midasSdkModule = await import("../../tests/utils/midasSdk");
+    const sdk = await midasSdkModule.getOrCreateMidas();
+
+    const signer = await ethers.getNamedSigner(taskArgs.creator);
+
+    const fuseFeeDistributor = (await ethers.getContract("FuseFeeDistributor", signer)) as FuseFeeDistributor;
+
+    const erc20PluginRewardsDel = await deployments.deploy("CErc20PluginRewardsDelegate", {
+      from: signer.address,
+      args: [],
+      log: true,
+      waitConfirmations: 1,
+    });
+    console.log("CErc20PluginRewardsDelegate: ", erc20PluginRewardsDel.address);
+
+    const erc20PluginRewardsDelegate = await ethers.getContract("CErc20PluginRewardsDelegate", signer);
+
+    const oldImplementations = [
+      ethers.constants.AddressZero,
```

Comment:
the first 3 values of the `oldImplementations` and `newImplementations` arrays are irrelevant and can be removed